reporters/loadReporter.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+var getReporter = require('jscs/lib/cli-config').getReporter;
+
+module.exports = function (reporter) {
+	// we want the function
+	if (typeof reporter === 'function') return reporter;
+
+	// load JSCS built-in or full-path or module reporters
+	if (typeof reporter === 'string' || !reporter) {
+		var rpt = getReporter(reporter).writer;
+		if (rpt) return rpt;
+		try {
+			return require(reporter);
```

Comment:
@sindresorhus I also thought it did a `require` at first, but take a second look at the [`getReporter` logic](https://github.com/jscs-dev/node-jscs/blob/7924d501200ebb81342000233faae53d28491519/lib/cli-config.js#L145-L155). As far as I can see, its logic is:

- If reporter argument is truthy
    - Try to find it relative to cwd
        - If it doesn't exist, `require()` it from JSCS's `reporters` dir
- Else (if reporter arg is falsy)
    - Default to a reporter inside of JSCS's `reporters` dir

There's no logic path that would `require` the reporter argument directly as far as I can see.
So should we keep this shortcut?